uaclient/tests/test_apt.py

```diff
@@ -347,140 +347,190 @@ def test_errors_on_apt_helper_process_timeout(
-        src_keyfile = os.path.join(KEYRINGS_DIR, "keyring")
-        dest_keyfile = os.path.join(APT_KEYS_DIR, "keyring")
-        gpg_export_calls = [mock.call(src_keyfile, dest_keyfile)]
         assert gpg_export_calls == m_gpg_export.call_args_list
 
+    @pytest.mark.parametrize("series", ("xenial", "noble"))
     @mock.patch("uaclient.apt.gpg.export_gpg_key")
     @mock.patch("uaclient.system.subp")
     @mock.patch("uaclient.apt.get_apt_auth_file_from_apt_config")
     @mock.patch("uaclient.apt.assert_valid_apt_credentials")
-    @mock.patch(
-        "uaclient.system.get_release_info",
-        return_value=mock.MagicMock(series="xenial"),
-    )
+    @mock.patch("uaclient.system.get_release_info")
     def test_add_auth_apt_repo_ignores_suites_not_matching_series(
         self,
         m_get_release_info,
         m_valid_creds,
         m_get_apt_auth_file,
         m_subp,
         m_gpg_export,
+        series,
         tmpdir,
     ):
         """Skip any apt suites that don't match the current series."""
         repo_file = tmpdir.join("repo.conf").strpath
         auth_file = tmpdir.join("auth.conf").strpath
         m_get_apt_auth_file.return_value = auth_file
-        # apt policy with xenial-updates enabled
+        # apt policy with series-updates enabled
         stdout = dedent(
             """\
-            500 http://archive.ubuntu.com/ xenial-updates/main amd64 \
+            500 http://archive.ubuntu.com/ {series}-updates/main amd64 \
                         Packages
-                release v=16.04,o=Ubuntu,a=xenial-updates,n=xenial,l=Ubuntu\
-                        ,c=main"""
+                release v=XX.04,o=Ubuntu,a={series}-updates,n={series},\
+                        l=Ubuntu,c=main""".format(
+                series=series
+            )
         )
         m_subp.return_value = stdout, ""
+        m_get_release_info.return_value = mock.MagicMock(series=series)
 
         add_auth_apt_repo(
             repo_filename=repo_file,
             repo_url="http://fakerepo",
             credentials="mycreds",
-            suites=("xenial-one", "xenial-updates", "trusty-gone"),
+            suites=(
+                "{}-one".format(series),
+                "{}-updates".format(series),
+                "trusty-gone",
+            ),
             keyring_file="keyring",
         )
 
-        expected_content = dedent(
-            """\
-            deb http://fakerepo xenial-one main
-            # deb-src http://fakerepo xenial-one main
-            deb http://fakerepo xenial-updates main
-            # deb-src http://fakerepo xenial-updates main
-        """
-        )
+        if series == "xenial":
+            expected_content = dedent(
+                """\
+                deb http://fakerepo xenial-one main
+                # deb-src http://fakerepo xenial-one main
+                deb http://fakerepo xenial-updates main
+                # deb-src http://fakerepo xenial-updates main
```

Comment:
And here we could use `{series}-` instead of hardcoded `xenial`. Just like `add_auth_apt_repo()` used above fo `suites=(...)`.